configs/platforms/debian-11-armhf.rb

```diff
@@ -0,0 +1,34 @@
+platform "debian-11-armhf" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "bullseye"
+
+  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends "
+
+  packages = [
+    "build-essential",
+    "make",
+    "quilt",
+    "pkg-config",
+    "debhelper",
+    "rsync",
+    "fakeroot",
+    "libbz2-dev",
+    "libreadline-dev",
+    "libselinux1-dev",
+    "make",
+    "openjdk-11-jre-headless",
+    "pkg-config",
+    "cmake",
+    "gcc",
+    "swig",
+    "systemtap-sdt-dev",
+    "zlib1g-dev"
+  ]
+
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive && apt-get update -qq && apt-get install -qy --no-install-recommends #{packages.join(' ')}"
+
+  plat.vmpooler_template "debian-11-armhf"
+  plat.output_dir File.join("deb", plat.get_codename, "PC1")
```

Comment:
I'm thinking `PC1` isn't needed as I don't see that in the arm64 version https://github.com/puppetlabs/puppet-runtime/blob/6a53e0224a2d3bd33e9d86152f37d2ec52d4f52f/configs/platforms/debian-11-amd64.rb#L33